platform/security/pom.xml

```diff
@@ -152,11 +152,6 @@
                 <version>${jasypt.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.shiro</groupId>
```

Comment:
correct, we could move it up to the root pom's `dependencymanagement` and define it once instead of in catalog and platform. I think it is fine the way it is though. Wouldn't save much.